uctMaths/competition/views.py

```diff
@@ -75,7 +76,7 @@ def printer_entry_result(request, school_list=None):
         if not (responsible_teacher or alt_responsible_teacher) and not school_list:
             return HttpResponseRedirect('../students/newstudents.html')
         #If the school has an entry
-        elif responsible_teacher:
+        elif responsible_teacher and request:
```

Comment:
Why did you split these branches the other branch still has ` c.update(csrf(request))`?

Reply:
I honestly can't remember why I did this, I'm testing removing this now